addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14100,7 +14100,25 @@ msgctxt "#36044"
 msgid "Disable polling for CEC capable devices?"
 msgstr ""
 
-#empty strings from id 36045 to 36100
+#: xbmc/peripherals/devices/PeripheralCecAdapter.cpp
+#: system/peripherals.xml
+msgctxt "#36047"
+msgid "Pause playback"
+msgstr ""
+
+#: xbmc/peripherals/devices/PeripheralCecAdapter.cpp
+#: system/peripherals.xml
+msgctxt "#36046"
+msgid "Keep audio, stop video"
+msgstr ""
+
+#: xbmc/peripherals/devices/PeripheralCecAdapter.cpp
+#: system/peripherals.xml
+msgctxt "#36045"
+msgid "No action"
+msgstr ""
+
+#empty strings from id 36048 to 36100
```

Comment:
I think you should reorder the ids, all others in this file are sorted ascending... ;-)